gateway-server/src/main/resources/createKnoxTokenDatabaseTable.sql

```diff
@@ -14,11 +14,12 @@
 --  the License.
 
 CREATE TABLE KNOX_TOKENS (
+   id varchar(128) NOT NULL,
    token_id varchar(128) NOT NULL,
    issue_time bigint NOT NULL,
    expiration bigint NOT NULL,
    max_lifetime bigint NOT NULL,
    username varchar(128),
    comment varchar(256),
-   PRIMARY KEY (token_id)
+   PRIMARY KEY (id)
```

Comment:
Thought: do you think using indexes e.g.`CREATE INDEX idx ON KNOX_TOKENS (token_id)` would speedup retrievals given we token ids are no longer PKs